database/migrations/2024_12_03_174541_create_flights_table.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+declare(strict_types=1);
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration
+{
+    public function up(): void
+    {
+        Schema::create('flights', function (Blueprint $table) {
+            $table->id();
+            $table->foreignIdFor(Lightit\Cities\Domain\Models\City::class, 'origin_city_id')->constrained()->cascadeOnDelete();
```

Comment:
Instead of:

Lightit\Cities\Domain\Models\City

You could do only 'City' and import the rest like you did with Migrations use Models\city;